client/less/main.less

```diff
@@ -840,6 +840,9 @@ code {
     .modal-footer {
       border: none;
     }
+    .modal-footer {
```

Comment:
Why are using this selector again?

Reply:
Do you want me to put this into: 
`.modal-header, .modal-footer{}`? 
Actually, putting values into the above selector will change the background of both the header and footer.